src/System.Diagnostics.Process/tests/ProcessCollectionTests.cs

```diff
@@ -12,7 +12,7 @@ public class ProcessCollectionTests : ProcessTestBase
         [Fact]
         public void TestModuleCollectionBehavior()
         {
-            ProcessModule[] mArray = _process.Modules.Cast<ProcessModule>().ToArray();
+            ProcessModule[] mArray = Process.GetCurrentProcess().Modules.Cast<ProcessModule>().ToArray();
```

Comment:
@Priya91, can you remind me how we'd end up in a situation where _process is a different bitness than Process.GetCurrentProcess()?